app/packages/looker/src/worker.ts

```diff
@@ -97,6 +97,7 @@ const processSample = ({ sample, uuid }: ProcessSample) => {
     ];
   }
 
+  mapId(sample);
```

Comment:
Hmm, I see. Are there any other places that need to be checked? Little bit scary.

Sorry about starting a waterfall of data representation changes via this JSON feature request (yes, data model improvement conversations noted)

Reply:
Yes, this is very much "for now" code. But, basically, the structure is the App itself still handles samples with `_id`, and within Looker everything is handled as `id`.

My bugs, didn't help. Giving it another pass now, but things look accounted for.